src/webview/js/js.js

```diff
@@ -867,6 +867,19 @@ documentBody.addEventListener('click', (e: MouseEvent) => {
     return;
   }
 
+  if (target.matches('.username')) {
+    const doc = target.closest('.message');
+    if (doc != null) {
+      sendMessage({
+        type: 'request-user-profile',
+        fromUserId: makeUserId(
+          requireNumericAttribute(doc.getElementsByClassName('avatar-img')[0], 'data-sender-id'),
```

Comment:
Rather than finding the `avatar-img` here, I'd add a `data-sender-id` attribute to the username element, and then you can just change the above if statement to `if (target.matches('.avatar-img') || target.matches('.username')) {`.

Reply:
I have added the suggested changes. Added the data-sender-id attribute in the messageAsHtml.js file. Also added updated snapshots for the tests.